ruby/lib/minitest/queue/runner.rb

```diff
@@ -314,7 +315,7 @@ def ordering_seed
       end
 
       def queue_url
-        @queue_url || ENV['CI_QUEUE_URL']
+        @queue_url || ENV['CI_QUEUE_URL'] || ENV['REDIS_URL'] || "redis://localhost:6379"
```

Comment:
`$REDIS_URL` is indeed a semi-standard, however it is much more likely `$REDIS_URL` will point a a Redis used by the tested app and not `ci-queue`.

It's actually a common setup error we got, people pointing ci-queue to the Redis instance dedicated to their test worker instead of the global one.

I'm also not sure defaulting to `localhost:6379` is a good idea, as it's very unlikely to be a working value.